src/engine/mega-macro-engine.lua

```diff
@@ -120,7 +120,7 @@ end
 local function GetMacroStubCode(macroId)
     return
         GenerateIdPrefix(macroId).."\n"..
-        "/click [btn:1] "..ClickyFrameName..macroId.."\n"..
+        "/click [btn:1] "..ClickyFrameName..macroId.." LeftButton\n"..
```

Comment:
With this line like this, keybinds dont work

Reply:
Can you let me know your client version? It works on mine, version 10.0.0.46313. 

A few people have mentioned this so I'm wondering if I'm missing a client update or if everyone else is >_>

Reply:
Same client as you have, playing in US realms

Reply:
Thanks for checking. That's very strange though, because if I revert that line, it fails for me. I need the 'LeftButton' added.

I'm going to sleep on it for now.

Reply:
Once this is resolved I'll get this in and released :)

Reply:
I just tried again with the new fork. It didn't work to start with. The commit didn't have the hook function commented out. Commenting out everything from line 739 onwards in mega-macro.window.lua and it works. 

The function seems to be for shift click adding spell names form the spell book. I'm guessing they've just updated the function name so will try to take a look at it later

Reply:
Works for me with no changes, live now, US.